doc/help/embed.md

```diff
@@ -0,0 +1,271 @@
+
+Here is an example of making a user a Programme Manager.
+
+![Make user a programme manager](user-as-programme-manager.png)
+
+This is what it looks like when you have successfully edited a user.
+
+![Successful Editing](successful-user-editing.png)
+
+# Programmes
+
+Programmes allow you to have multiple projects in with one overview
+and leadership team. Project Managers and users on projects can only
+see data in their projects allowing for privacy between project teams,
+but an overview for the programme team.
+
+## Before Adding
+
+![Programmes Listing](programmes-listing.png)
+
+## Adding
+
+![Editing Programmes](edit-programmes.png)
+
+## After Adding
+
+![After adding programme](after-adding-programme.png)
+
+## Viewing
+
+![Programme Details](programme-detail.png)
+
+# Projects
+
+Projects are a way of getting people together to work on a group of
+properties. All data in a project are editable by a Programme or
+Project Manager. Users in a project can view all private data in that
+project.
+
+## Adding a Project
+
+Projects must have a project name.
+
+![Adding a project](adding-project.png)
+
+## Listing Projects
+
+![Listing projects](projects-listing.png)
+
+## Viewing project details
+
+![Project details](project-details.png)
+
+# Uploading Property Details
+
+You can upload a csv file to create a property with profile
+details.
+
+![Property upload](add-properties.png)
+
+To make it easier to create the csv an Excel template has been
+created. To use it, fill in the Excel spreadsheet as below and save
+the file as csv when you are done. The csv can be uploaded as in the
+picture above.
+
+## Property Details Form
+
+[This Excel spreadsheet](example-property.xlsx) has been designed to
+comprehensively capture the diverse range of building characteristics
+(house type, construction, technologies installed); plus very
+top-level info on the initial design intention and final output. The
+information in this form will be fed directly into the Embed database
+associated with each dwelling with the intention of giving external
+parties a clear view on the home, technologies installed and the
+intended/achieved performance.
+
+Information is required from the dwelling as designed, and from the
+dwelling as built, as these specifications may be different. There are
+also opportunities to add interventions that identify times when the
+performance of the dwelling may change. This could be an intervention
+in the fabric, service or a change in occupation.
+
+There are many data fields for completion, but not all will be
+relevant to your study building(s). Please attempt to complete all
+relevant fields to the best of your ability.
+
+As far as is possible fields have been pre-populated with information
+for you to select, however we fully expect there to be omissions and
+encourage the giving of additional info in boxes that request either
+'specify or give further information' boxes. These fields should only
```

Comment:
" boxes that request either 'specify or give further information'" - this sentence looks like it's missing a couple of words?